components/nodes/NewsletterLinkNode.js

```diff
@@ -0,0 +1,10 @@
+import { Paragraph, Anchor } from '../common/CommonStyles';
```

Comment:
Is this file used? I can't find any reference to it.

Reply:
You're right, neither Newsletter*Node.js files are used, I've removed them.